### ISSUE TITLE
DLL built with mingw on Linux crashes editor

### ISSUE THREAD

**Issue by ianling** [godotengine/godot-git-plugin#148]
I am building a Windows DLL from my Linux machine using mingw. The build succeeds without errors, but when I copy the DLL over to my Windows machine and open Godot 4 beta10, the editor crashes immediately upon opening my project. If I use the official DLL from the latest release on this Github repo, the project opens fine.

Am I missing a step somewhere?

Windows host: Windows 10 x86_64

Linux host: Ubuntu x86_64, mingw-w64 v8.0.0-1, g++-mingw-w64 v10.3.0-14ubuntu1+24.3

**Comment by ianling** [comment]
I think this may be due to certain libraries not being statically included in the DLL. I was able to get it working by copying several additional DLLs into my Godot project.

* libgcc_s_seh-1.dll
* libstdc++-6.dll
* libwinpthread-1.dll

**Comment by markeel** [comment]
This doesn't feel like a plugin bug, if anything it feels like a documentation issue for mingw64 builds.  Or is the assumption that static linking should be done and these are particular to this plugin?

**Comment by ianling** [comment]
Agreed, regardless of what way ultimately gets recommended, all I'm trying to call out here is that some additional documentation for building with mingw64 would be nice
